ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -20,6 +20,7 @@ import "google/protobuf/duration.proto";
 import "com/digitalasset/daml_lf_dev/daml_lf.proto";
 import "com/digitalasset/daml/lf/transaction.proto";
 import "com/digitalasset/daml/lf/value.proto";
+import "ledger/participant-state/protobuf/ledger_configuration.proto";
```

Comment:
Please take a look at the ledger-api protobuf definitions and accompanying build rules to avoid having to prefix the path in the repository:
https://github.com/digital-asset/daml/blob/1bd2ec5b1a441e3a933b99e3d0473b6b64a027af/ledger-api/grpc-definitions/BUILD.bazel#L16-L21

Reply:
better way to do that is to use relative path like we do in transaction.proto:
https://github.com/digital-asset/daml/blob/master/daml-lf/transaction/BUILD.bazel#L19